### PR TITLE
Query multifield benchmark added

### PR DESCRIPTION
**Pull request by deepakdinesh1123**

### CODE REVIEW

**Comment by deepakdinesh1123** [comment]
Some of the benchmarks that I have written are query-related and require some of the changes made in this PR so I will be adding them after this gets merged.

**Comment by smithdc1** [comment]
Hey @deepakdinesh1123  -- I've rebased, applied black, isort and checked for flake8 errors. This will need to be done for the other PRs too, could you start to work on those?

File: benchmarks/query_benchmarks/query_multifield/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from .models import MultiField
+
+
+class QueryMultiField:
+    def setup(self):
+        bench_setup(migrate=True)
+        for i in range(0, 3000):
+            kwargs = {}
+            for j in range(1, 11):
+                kwargs["field%s" % j] = "foobar_%s_%s" % (i, j)
```
Comment (smithdc1) [inline benchmarks/query_benchmarks/query_multifield/benchmark.py]: Could we rewite as a comprehension? Is this equivilent? 

```suggestion
            kwargs = {f"field%s" : f"foobar_{i}_{j}" for j in range(1,11)}
```

**Comment by deepakdinesh1123** [comment]
isort check is failing but the error message I am getting on my machine and the action is very different I had a similar problem yesterday where isort was not throwing an error on my machine but the action was failing.

**Comment by deepakdinesh1123** [comment]
isort check is failing with an error but I am not getting any error on my machine. I tried to configure known_first_party and other options but to no avail.

**Comment by smithdc1** [comment]
I'll have a look at [boosting our developer experience](https://adamj.eu/books/#boost-your-django-dx) and use pre-commit / pre-commit ci so lint errors get automatically corrected.

**Pull request merged**